defaultPasswords.json

```diff
@@ -2,5 +2,6 @@
   "secretpassword": "stillasecret",
   "gmpass": "stillasecret",
   "emailuser": "stillasecret@example.com",
-  "emailpass": "stillasecret"
+  "emailpass": "stillasecret",
+  "FB_API_KEY": "311861065837235|259a4afadcef21e883b1c927541af836"
```

Comment:
Please delete this! This information should never be exposed! (FB_API_KEY) Thanks!